src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
```

Comment:
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => (
  <div className="CommentInfo">
```
один батьківський компонент, цей темплейт не потрібен)